.spelling

```diff
@@ -1,48 +1,42 @@
-SourceBots
+100x100mm
+11.1v
+20000hz
+20hz
+2200mah
+2.2a.
+250x250mm
+2mm
+3.3v
+3.81mm
+3m
+4mm
+5mm
+70x84x20mm
+7.5mm
+83x99x24mm
+apriltag
 arduino
-pullup
+booleans
 camcon
-microswitches
-WiFi
-11.1V
-H0-1
-L0-3
-7.5mm
-5mm
+camcons
+codecademy
+dropdown
 enum
-booleans
-Codecademy
-LearnPython
+h0-1
+imax
+intellisense
+l0-3
+learnpython
+lipo
+macos
+microswitches
+numpy
+piezo
+pycharm
 python3
-dropdown
-3.81mm
-camcons
-micro-usb
-Smallpeice
-LiPo
+scipy
+sourcebots
```

Comment:
The spellings file is case sensetive. If an entry isnt in lower case, then that case must be enforced in the doc. I suggest forcing _SourceBots_ casing, and probably on a few others too

Reply:
Wasn't aware of that, thanks. I'll leave the case of spelling entries as they are.